packages/app/src/vis-packs/core/complex/config.tsx

```diff
@@ -13,7 +13,9 @@ interface ComplexConfig {
 function createStore() {
   return create<ComplexConfig>(
     persist(
-      (set) => ({
+      // https://github.com/pmndrs/zustand/issues/701
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      (set, get) => ({
```

Comment:
zustand has refactored the middleware typings, but there are still a few glitches: https://github.com/pmndrs/zustand/issues/701